src/Apps/ViewingRoom/__tests__/ViewingRoomsApp.jest.tsx

```diff
@@ -41,7 +41,7 @@ describe("ViewingRoomsApp", () => {
             allViewingRooms: viewer {
               ...ViewingRoomsApp_allViewingRooms
             }
-            featuredViewingRooms: viewingRooms(featured: true) {
+            featuredViewingRooms: viewingRoomsConnection(featured: true) {
```

Comment:
does this need any pagination/`first: 10` type args?

Reply:
It doesn't _require_ it (gravity returns 20 items), you think I should explicit param?

Will merge and if your answer will be yes - will create another PR